project_resource_calendar/models/calendar_event.py

```diff
@@ -116,6 +126,14 @@ class CalendarEvent(models.Model):
         'Current ID',
     )
 
+    @api.one
```

Comment:
@api.depends('allow_double_book') or @api.depends('room_id'), right?.. so it is re computed when the fields that this one depends on one change

Reply:
Yes I added it, when we change the room_id the equipments changes also